build.gradle

```diff
@@ -37,36 +42,24 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
-/*configure(subprojects.findAll { !it.name.contains("testpack") && !it.name.startsWith("module") }) {
+configure(subprojects.findAll { !it.name.contains("testpack") && !it.name.startsWith("module") && !it.name.contains("android")}) {
+    // General Java configuration
+    apply plugin: 'java'
     apply plugin: 'checkstyle'
     apply plugin: 'pmd'
     apply plugin: 'findbugs'
-    apply plugin: 'maven'
-    apply plugin: 'java-library-distribution'
-    apply plugin: 'artifactory-publish'
+    apply plugin: 'com.jfrog.artifactory'
     apply plugin: 'maven-publish'
 
-    // Primary dependencies definition
-    dependencies {
-        compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.21'
-        compile group: 'com.google.guava', name: 'guava', version: '19.0'
-
-        // These dependencies are only needed for running tests
-        testCompile group: 'junit', name: 'junit', version: '4.12'
-        testCompile group: 'ch.qos.logback', name: 'logback-classic', version: '1.1.7'
-        testCompile group: 'org.mockito', name: 'mockito-core', version: '1.10.19'
-    }
-
     // Set the expected module Java level (can use a higher Java to run, but should not use features from a higher Java)
     sourceCompatibility = 1.8
     targetCompatibility = 1.8
 
     jar {
         manifest {
-            attributes("Implementation-Title": project.name,
-                    "Implementation-Version": version)
+            attributes("Implementation-Title": project.name, "Implementation-Version": version)
         }
-        from(['LICENSE', 'NOTICE'])
```

Comment:
I probably should at a NOTICE file.

Reply:
Yeah not sure where it went if we had one, or if not why this was in here in the first place. Copy paste ftw maybe :-)